codes/data/multi_country/quality-check-processed-datasets.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+import numpy as np
+from codes import utils as cutil
+
+#### Settings
+
+path_cutoff_dates = cutil.HOME / 'codes' / 'data' / 'cutoff_dates.csv'
+cutoff_dates = pd.read_csv(path_cutoff_dates)
```

Comment:
Just for good practice, lets wrap everything from here down thats not just defining a variable (like `use_cutoff`) and isn't already in a function with a `main()` function and use the syntax:
```
if __name__=="__main__":
    main()
```

Just so that people could in theory import stuff from this file into other scripts without running the whole script

Reply:
Ok I did this, mostly, but let me know if there are some things that should be moved into or back out of `main()`. Thanks!